modules/gltf/src/lib/types/gltf-json-schema.ts

```diff
@@ -1083,6 +1084,8 @@ type FeatureTexture = {
   extras?: any;
   [key: string]: any;
 };
+export type {FeatureTexture as EXT_feature_metadata_feature_texture};
```

Comment:
- Can we have these exports at the bottom together with the other exports?
- Can we minimize the arbitrary renaming of things? - why do these need to be renamed where as other extension exports do not?